akka-stream/src/main/scala/akka/stream/Materializer.scala

```diff
@@ -31,6 +35,24 @@ abstract class Materializer {
    */
   implicit def executionContext: ExecutionContextExecutor
 
+  /**
+   * Interface for stages that need timer services for their functionality. Schedules a
+   * single task with the given delay.
+   *
+   * @return A [[akka.actor.Cancellable]] that allows cancelling the timer. Cancelling is best effort, if the event
+   *         has been already enqueued it will not have an effect.
+   */
+  def scheduleOnce(delay: FiniteDuration, task: Runnable): Cancellable
+
+  /**
+   * Interface for stages that need timer services for their functionality. Schedules a
+   * repeated task with the given interval between invocations.
+   *
+   * @return A [[akka.actor.Cancellable]] that allows cancelling the timer. Cancelling is best effort, if the event
+   *         has been already enqueued it will not have an effect.
+   */
+  def schedulePeriodically(interval: FiniteDuration, task: Runnable): Cancellable
+
```

Comment:
No, I meant adding it to the GraphStageLogic, i.e. "overload" just not with the same name